02week/pigLatin.js

```diff
@@ -11,7 +11,29 @@ const rl = readline.createInterface({
 function pigLatin(word) {
```

Comment:
@LuisCarter8791 
your pigLatin function isn't passing any tests, please take a look and fix this if you want credit for the pigLatin assignment, thank you!